README.md

```diff
@@ -68,7 +68,7 @@ For example, in an initializer:
 // app/initializers/register-showdown-extensions.js
 
 export function initialize() {
-  window.showdown.extension("myExtensionName", function() {
+  showdown.extension("myExtensionName", function() {
```

Comment:
Can no longer reference window as that won't work in node land obviously.  Just always grab off the global context